htmresearch/support/csv_helper.py

```diff
@@ -51,17 +51,23 @@ def readCSV(csvFile, numLabels=0):
         sampleIdx = headers.index("Sample")
         idIdx = headers.index("ID")
       except ValueError as e:
-        print ("Could not find \'ID\' and/or \'Sample\' columns, so assuming "
+        print ("Could not find 'ID' and/or 'Sample' columns, so assuming "
                "they are 0 and 2, respectively.")
         sampleIdx = 2
         idIdx = 0
-      labelIdx = range(sampleIdx + 1, sampleIdx + 1 + numLabels)
-
+      
       dataDict = OrderedDict()
-      for lineNumber, line in enumerate(reader):
-        dataDict[lineNumber] = (line[sampleIdx],
-                                [line[i] for i in labelIdx if line[i]],
-                                line[idIdx])
+
```

Comment:
It was necessary in the previous versions of the NLP models. So yes, a normal dict would be better (simpler).